consensus/hotstuff/pacemaker/view_tracker_test.go

```diff
@@ -0,0 +1,254 @@
+package pacemaker
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/helper"
+	"github.com/onflow/flow-go/consensus/hotstuff/mocks"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func TestViewTracker(t *testing.T) {
+	suite.Run(t, new(ViewTrackerTestSuite))
+}
+
+type ViewTrackerTestSuite struct {
+	suite.Suite
+
+	initialView uint64
+	initialQC   *flow.QuorumCertificate
+	initialTC   *flow.TimeoutCertificate
+
+	livenessData *hotstuff.LivenessData // Caution: we hand the memory address to viewTracker, which could modify this
```

Comment:
let's just remove it from member section, and we won't need to think about safety. It's only used to build mock of persister for initial call of `GetLivenessData`.

Reply:
👍 👉  https://github.com/onflow/flow-go/pull/4155/commits/ddcd4a65aedcc49bdd608ae8bf513161107f6a61